libr/anal/p/anal_bpf.c

```diff
@@ -59,15 +58,15 @@ static int disassemble(RAnalOp *r_op, ut64 pc, const ut8 *buf, int len) {
 		break;
 	case BPF_LD_W | BPF_IND:
 		op = r_bpf_op_table[BPF_LD_W];
-		fmt = "[x+%d]";
+		fmt = "[x%+d]";
 		break;
 	case BPF_LD_H | BPF_IND:
 		op = r_bpf_op_table[BPF_LD_H];
-		fmt = "[x+%d]";
+		fmt = "[x%+d]";
```

Comment:
wtf?

Reply:
@trufae `%+d` is a valid format string. This fixes a bug where negative offsets get printed as `[x+-5]` instead of `[x-5]`.

Reply:
correct :) sorry for the confussion